compiler/src/iree/compiler/Codegen/Common/test/strip_compilation_info.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: iree-opt --split-input-file --iree-codegen-strip-compilation-info %s | FileCheck %s
+
+#translation_info = #iree_codegen.translation_info<LLVMGPUVectorDistribute workgroup_size = [64, 1, 1] subgroup_size = 64>
+func.func @main() attributes {translation_info = #translation_info} {
+    return
+}
+
+// CHECK-LABEL: func.func @main
+// CHECK-NOT:   #translation_info =
```

Comment:
Most of the checks in this file seem off: there's no `#` symbol on the attribute name